navbar-service.js

```diff
@@ -28,6 +28,15 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  // service.display may be a string or an array of strings
+  // valid values: all, brand, menu, toolbar
+  service.display = 'all';
+  service.siteTitle = null;
+
+  $rootScope.$on("$routeChangeSuccess", function() {
```

Comment:
Probably overly complicated. We should maybe just have a default setting that you can change site wide that starts of using `all`. Yes, that's better.